test/tests/partitioners/single_rank_partitioner/tests

```diff
@@ -0,0 +1,24 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'single_rank_partitioner.i'
+    exodiff = 'single_rank_partitioner_out.e'
+    min_parallel = 4
```

Comment:
```suggestion
```

Reply:
No - the min parallel is on purpose... the test puts the solve on rank 2

Reply:
Fine fine. but why not 2 then?